src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7106,6 +7110,12 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
         })
         print("The snapshot resource id %s " % snapshot_resource_id)
 
+        # update tags on nodepool snapshot
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=75629451&view=results) to validate the change.

Reply:
Live test passed, CI failed due to the recording file you've committed is not correct. May commit the recording file from pipeline artifact `live test reports/ext-recordings/test_aks_nodepool_snapshot.yaml`